config/src/config.rs

```diff
@@ -299,8 +309,8 @@ pub struct Mining {
     /// Set to 0 to disable timeouts.
     #[partial_struct(serde(
         default,
-        deserialize_with = "from_secs",
-        rename = "data_request_timeout_seconds"
+        deserialize_with = "from_millis",
+        rename = "data_request_timeout_millis"
```

Comment:
`millis` sounds strange to me
```suggestion
        rename = "data_request_timeout_ms"
```

Reply:
It doesn't feel natural to me either, but it is the convention that `Duration` is using.

Anyway, I support the idea that the convention of a library can affect our code for the sake of ease of understanding — but it should never affect the configuration files, where we need to use whatever gives better UX.

My only objection to `data_request_timeout_ms` is that using an acronym here (`ms`) sort of feels out of place among the rest of settings, which use full magnitude names like `seconds`.